server/py/services/api/initial_data.py

```diff
@@ -993,6 +1002,46 @@ def _create_project_summaries(db, db_session):
     db._upsert(db_session, project_summaries, ignore=True)
 
 
+def _init_system_id(db_session: sqlalchemy.orm.Session):
+    # Initializes a system id for MLRun deployment.
+    # The system id is first checked in the database. If it does not exist, the function checks for a configured id
+    # (from environment variables or config), and if neither is found, a new random one is generated and stored.
+    db = framework.db.sqldb.db.SQLDB()
+
+    # check if a system id already exists in the database
+    system_id = db.get_system_id(db_session)
+
+    if system_id is not None:
+        logger.info("Existing system id found in the database", system_id=system_id)
+        return
+
+    # check if the system id is configured via environment variables or configmap
+    system_id = _get_configured_system_id()
+
+    if system_id is not None:
+        logger.info("Using configured system id", system_id=system_id)
+        db.create_system_id(db_session, system_id)
+        return
+
+    # if no system id is found, generate a new one
+    system_id_str = _generate_system_id()
+    db.create_system_id(db_session, system_id_str)
+
+
+def _get_configured_system_id() -> typing.Optional[str]:
+    system_id = os.environ.get("SYSTEM_ID")
```

Comment:
This env var is not something we support in MLRun.
Env vars are translated to the MLRun config [automatically](https://github.com/mlrun/mlrun/blob/94c3f48d248608872fde6a0e4882f6c5254bf9a4/mlrun/config.py#L1411-L1413), when they have a certain structure:
```
MLRUN_SYSTEM_ID
```
This will env var will be translated to `mlrun.mlconf.system_id`, so no need to manually check the env var here.